src/Command/DkanCommands.php

```diff
@@ -43,22 +68,24 @@ public function dkanTestCypress(array $args)
           $cypress->arg($arg);
         }
 
-        return $cypress->run();
+        $cypress->run();
+        return $this->deleteTestUsers();
     }
 
     /**
      * Run DKAN Dredd Tests.
      */
     public function dkanTestDredd()
     {
-        $this->apiUser();
+        $this->createTestUsers();
         $this->taskExec("npm install dredd")
             ->dir("docroot/modules/contrib/dkan")
             ->run();
 
-        return $this->taskExec("npx dredd --hookfiles=./dredd-hooks.js")
+        $this->taskExec("npx dredd --hookfiles=./dredd-hooks.js")
             ->dir("docroot/modules/contrib/dkan/dredd")
             ->run();
+        return $this->deleteTestUsers();
```

Comment:
```suggestion
        $this->deleteTestUsers();
```
The return statement was a bit confusing to me since `deleteTestUsers()` doesn't seem to return anything. I think it's a bit more readable without it.